### PR TITLE
v1.16 release

### PR DESCRIPTION
**Pull request by Sjors**
To be merged after production deploy.

### CODE REVIEW

File: tests/services/export-history.service.spec.coffee [hunk 1]
```diff
@@ -0,0 +1,77 @@
+describe "ExportHistory service", ->
```
Comment (Sjors) [inline tests/services/export-history.service.spec.coffee]: Nice, thanks for extracting these.

File: tests/controllers/settings_change_second_password_ctrl_spec.coffee [hunk 2]
```diff
@@ -79,32 +84,45 @@ describe "ManageSecondPasswordCtrl", ->
...
       expect(scope.form.password.$valid).toEqual(false)
       expect(scope.form.password.$error.isValid).toEqual(true)
 
     it "should not be valid if the password is the main password", ->
       scope.form.password.$setViewValue('mainpw')
+      $timeout.flush()
+      expect(scope.form.password.$valid).toEqual(false)
+      expect(scope.form.password.$error.isValid).toEqual(true)
+
+    it "should not be valid if incorrect sec pass is entered when removing", ->
```
Comment (Sjors) [inline tests/controllers/settings_change_second_password_ctrl_spec.coffee]: This test fails on my local machine on the branch for #1090. Any idea what could have broken it?

Comment (jtormey) [inline tests/controllers/settings_change_second_password_ctrl_spec.coffee]: Hmm not sure, will check it out.

Comment (Sjors) [inline tests/controllers/settings_change_second_password_ctrl_spec.coffee]: I haven't seen this again since this morning.

**Pull request merged**